daffodil-cli/src/it/scala/org/apache/daffodil/tunables/TestCLITunables2.scala

```diff
@@ -34,13 +34,13 @@ class TestCLITunables2 {
 
     try {
       // note: 2>&1 is shell-speak for "connect stderr into stdout"
-      val cmd = String.format("""echo "a,b" | %s parse -s %s -TsuppressSchemaDefinitionWarnings=all 2>&1""", Util.binPath, testSchemaFile)
+      val cmd = String.format("""echo a,b| %s parse -s %s -TsuppressSchemaDefinitionWarnings=all 2>&1""", Util.binPath, testSchemaFile)
```

Comment:
Shouldn't this also be Util.echo(...)

Reply:
For the record, I concur with Steve. The 'Util.echoN' function is only intended to cover the difference in 'echo' behavior 
 between Windows and Linux when the stated conditions are met.